state/src/applicator/task_queue.rs

```diff
@@ -50,6 +54,23 @@ fn new_running_state() -> QueuedTaskState {
     QueuedTaskState::Running { state: PENDING_STATE.to_string(), committed: false }
 }
 
+/// Record the length of a task queue
+fn record_task_queue_length<T: TransactionKind>(key: &TaskQueueKey, tx: &StateTxn<'_, T>) {
```

Comment:
compiler is _probably_ smart enough to do this, but i'd `#[inline]` this to avoid a function call when the feature is disabled